dev/push-images

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -x
+
+source $(git rev-parse --show-toplevel)/dev/tasks/common
+
+dev/tasks/push-cloud-controller-manager
```

Comment:
there are several images here, see https://github.com/kubernetes/cloud-provider-gcp/blob/master/cloudbuild.yaml

Reply:
is ok if you want to add TODOs, no need to do it right now but I expect the other projects to use the same script

Reply:
Added a second script for the other image (gcp-controller-manager).  It's a little annoying that we have to set all those ldflags twice, but I think still clearer than trying to abstract this (at least, I don't have any great ideas that aren't way too complicated for N=2).

Reply:
Actually, figured out a nice way, because ko can use env vars.  Created a .ko.yaml file in the root.